site/_docs/reference.md

```diff
@@ -2724,6 +2724,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | FLOOR(value)                                   | Similar to standard `FLOOR(value)` except if *value* is an integer type, the return type is a double
 | b | FORMAT_DATE(string, date)                      | Formats *date* according to the specified format *string*
 | b | FORMAT_DATETIME(string, timestamp)             | Formats *timestamp* according to the specified format *string*
+| h s | FORMAT_NUMBER(value, decimalOrFormat)        | Formats the number *value* like '#,###,###.##', rounded to decimal places. If *decimalOrFormat* is 0, the result has no decimal point or fractional part. The *decimalOrFormat* also accept a user specified format. This is supposed to function like MySQL's FORMAT
```

Comment:
I am wondering if it makes sense to split this into two entries, one for `FORMAT_NUMBER(value, decimal)` and another for `FORMAT_NUMBER(value, format)`. I think it would make writing the descriptions easier.

Reply:
I agree with you. It will be more readable for user.